smtp/smtpc.c

```diff
@@ -241,6 +241,14 @@ smtpc_request_new(struct smtpc_env *env, const char *url, const char *from)
 		return NULL;
 	}
 	memset(req, 0, sizeof(*req));
+	req->error = (char *) malloc(CURL_ERROR_SIZE + sizeof(" (SMTP error)"));
+	if (req->error == NULL) {
+		free(req);
+		box_error_set(__FILE__, __LINE__, ER_MEMORY_ISSUE,
+					  "Can't alloc error message buffer");
+		return NULL;
+	}
+	memset(req, 0, sizeof(*req->error));
```

Comment:
I guess result will be wrong. `sizeof(*req->error)` == `sizeof(char)` (not whole string).

Reply:
Fixed.